src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificatesTabCoordinator.swift

```diff
@@ -281,6 +281,9 @@ final class HealthCertificatesTabCoordinator {
 			presentCovPassInfoScreen: { [weak self] viewController in
 				self?.presentCovPassInfoScreen(rootViewController: viewController)
 			},
+			onDismiss: { [weak self] in
+				self?.overviewScreen.tableView.reloadData()
+			},
```

Comment:
Not needed if you use the same `objectDidChange` mechanism that we use for `isNewBoosterRule` and `isNewCertificateReissuance`